### PR TITLE
flips: add page

### PR DESCRIPTION
**Pull request by Fabiokleis**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
Flips v1.31

### CODE REVIEW

File: pages/common/flips.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# flips
+
+> Flips Applies and creates IPS and BPS patches.
+> More information: <https://www.smwcentral.net/?a=details&id=11474&p=section>.
```
Comment (navarroaxel) [inline pages/common/flips.md:4]: I think the source code is more useful
```suggestion
> More information: <https://github.com/Alcaro/Flips>.
```

File: pages/common/flips.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Open user interface to create/apply patch:
+
+`flips`
+
+- Apply patch by creating new file:
+
+`flips --apply {{patch.bps}} {{rom.smc}} {{hack.smc}}`
+
+- Create patch from two roms:
```
Comment (navarroaxel) [inline pages/common/flips.md:14]:
```suggestion
- Create patch from two ROMs:
```

File: pages/common/flips.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# flips
+
+> Flips Applies and creates IPS and BPS patches.
+> More information: <https://www.smwcentral.net/?a=details&id=11474&p=section>.
+
+- Open user interface to create/apply patch:
+
+`flips`
+
+- Apply patch by creating new file:
```
Comment (navarroaxel) [inline pages/common/flips.md:10]: Perhaps this better
```suggestion
- Apply a patch and create a new ROM file:
```

File: pages/common/flips.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# flips
+
+> Flips Applies and creates IPS and BPS patches.
```
Comment (navarroaxel) [inline pages/common/flips.md:3]: Perhaps this better
```suggestion
> Create and apply patches for IPS and BPS files.
```

File: pages/common/flips.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://www.smwcentral.net/?a=details&id=11474&p=section>.
+
+- Open user interface to create/apply patch:
```
Comment (navarroaxel) [inline pages/common/flips.md:6]:
```suggestion
- Open to create and apply patches interactively:
```

File: pages/common/flips.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# flips
+
+> Create and apply patches for IPS and BPS files.
+> More information: <https://github.com/Alcaro/Flips>.
+
+- Open to create and apply patches interactively:
```
Comment (navarroaxel) [inline pages/common/flips.md:6]: Perhaps this better
```suggestion
- Start Flips to create and apply patches interactively:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/flips.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+- Start Flips to create and apply patches interactively:
+
+`flips`
+
+- Apply a patch and create a new ROM file:
+
+`flips --apply {{patch.bps}} {{rom.smc}} {{hack.smc}}`
+
+- Create patch from two ROMs:
```
Comment (kbdharun) [inline pages/common/flips.md:14]:
```suggestion
- Create a patch from two ROMs:
```

File: pages.pt_BR/common/flips.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# flips
+
+> Flips Cria e aplica patches em arquivos IPS e BPS.
+> Mais informações: <https://github.com/Alcaro/Flips>.
+
+- Abre o flips em uma interface para criar e aplicar um patch:
```
Comment (navarroaxel) [inline pages.pt_BR/common/flips.md:6]: Perhaps
```suggestion
- Abre Flips para criar e aplicar um patch:
```

File: pages.pt_BR/common/flips.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
+# flips
+
+> Flips Cria e aplica patches em arquivos IPS e BPS.
```
Comment (navarroaxel) [inline pages.pt_BR/common/flips.md:3]:
```suggestion
> Cria e aplica patches em arquivos IPS e BPS.
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**